_test_/pet.test.js

```diff
@@ -1,11 +1,149 @@
 const Pet = require('../src/pet');
 
 describe('constructor', () => {
+  it('returns instance of an object', () => {
+    expect(new Pet('Romy')).toBeInstanceOf(Object);
+  });
+
   it('sets the name property', () => {
-    const pet = new Pet('Fido');
-    expect(pet.name).toEqual('Fido');
-    
     const romy = new Pet('Romy');
     expect(romy.name).toEqual('Romy');
   });
+
+  it('sets the initial age', () => {
+    const romy = new Pet('Romy');
+    expect(romy.age).toBe(0);
+  });
+
+  it('sets the growth in increament of 1', () => {
+    const romy = new Pet('Romy');
+
+    romy.growUp();
+
+    expect(romy.age).toBe(1);
+    expect(romy.hunger).toBe(5);
+    expect(romy.fitness).toBe(7);
+  });
+
+  it('increases fitness by 4', () => {
+    const romy = new Pet('Romy');
+
+    romy.fitness = 4;
+    romy.walk();
+
+    expect(romy.fitness).toEqual(8);
+  });
+
+  it('increases fitness by 8 to a maximum of 10', () => {
+    const romy = new Pet('Romy');
+
+    romy.fitness = 8;
+    romy.walk();
+
+    expect(romy.fitness).toEqual(10);
+  });
+
+  it('decreases hunger level by 3', () => {
+    const romy = new Pet('Romy');
+
+    romy.hunger = 8;
+    romy.feed();
+
+    expect(romy.hunger).toEqual(5);
+  });
+
+  it('to check if romy needs a walk', () => {
+    const romy = new Pet('Romy');
+
+    romy.fitness = 3;
+    romy.checkUp();
```

Comment:
you call the function once here, but then call it again in your expect block, you only need to call it once, in the expect block.